src/components/views/right_panel/RoomSummaryCard.tsx

```diff
@@ -199,6 +200,8 @@ const RoomSummaryCard: React.FC<IProps> = ({ room, onClose }) => {
     };
 
     const isRoomEncrypted = useIsEncrypted(cli, room);
+    const roomContext = useContext(RoomContext);
```

Comment:
I guess I am missing how the E2E status gets into this context, but presumably you have tested it...

Reply:
I have tested it. No idea how it works, and no idea how the code I copy/pasted works.

Reply:
> I guess I am missing how the E2E status gets into this context, but presumably you have tested it...

The context contains the full RoomView state (IState)
See RoomView doing `<RoomContext.Provider value={this.state}>`